docs/tutorials/Fundamentals.md

```diff
@@ -52,12 +61,15 @@ Regarding data formats, iTowns offers several possibilities : [vector tile](http
 It was earlier mentioned that data are displayed as [`Layers`]{@link Layer} within iTowns. 
 Several specific types of `Layers` exist, the use of which depends on the data to display :
 
-- `{@link ColorLayer}` can be used to display raster graphics or vector data projected on the ground (left picture in the table bellow),
+- `{@link ColorLayer}` can be used to display raster graphics or vector data that needs to be rasterized to be projected on the ground (left picture in the table bellow),
 - `{@link ElevationLayer}` can be used to display 3D elevation models (center picture in the table bellow),
-- `{@link GeometryLayer}` can be used to display 3D objects, such as geometric shapes or buildings modelling (right picture in the table bellow).
-- `{@link FeatureGeometryLayer}` is a pre-configured `{@link GeometryLayer}` which simplifies its implementation in given cases.
-- `{@link PointCloudLayer}` can be used to display 3D point clouds.
-- `{@link OrientedImageLayer}` can be used to display some oriented images.
+- `{@link GeometryLayer}` can be used to display 2D vector data or 3D objects, such as geometric shapes or buildings modelling (right picture in the table bellow).
+- `{@link FeatureGeometryLayer}` is a pre-configured `{@link GeometryLayer}` which simplifies its implementation in given cases (e.g. to extrude 3D buildings).
+- `{@link PointCloudLayer}` can be used to display 3D point clouds. Any point cloud formats are supported as long as the corresponding parser is provided.
```

Comment:
Do you think it would be relevant to speak of `Source` instead of `parser` ?

Reply:
yep, updated